### DISCUSSION TITLE
Display more than 10 logs at one click in Loki's Log Row Context Viewer

### DISCUSSION THREAD

**Discussion by yandingkang** [grafana/grafana#43276] [Observability]
**Steps to reproduce**:
1.Open Grafana Explore
2.Select Loki datasource
3.Type a query with key word
4.Click "show context" button
5.Click "Load 10 more" button
**Actual Result**:
Display 10 surrounding logs of the log line I am investigating.
**Expected Result**
Manually set the number of lines to display,for example 100 lines.
**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: Grafana v8.2.1 (88622d7)
- Data source type & version:Loki,v2.3.0
- User OS & Browser:Win10,Chrome
- Others:
![image](https://user-images.githubusercontent.com/76470592/146517907-725ac589-e043-4d2d-8df0-def99333ee65.png)

**Comment by hunkjun** [comment]
more function： can find or filter is better in ”show context“.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
